README.md

```diff
@@ -20,16 +22,89 @@ Or install it yourself as:
 
     $ gem install pmc_miller
 
+
 ## Usage
 
-TODO: Write usage instructions here
+### Reader
+
+The `PmcMiller::Reader` class can be used to accumulate service data for a
+given key.  This provides a ruby object based approach to
+[grepping metrics](https://github.com/puppetlabs/puppetlabs-puppet_metrics_collector#grepping-metrics).
+
+The following code snippet initializes the reader object to a specified
+`puppet-metrics-collector` path and then accumulates the data for `queue_depth`
+associated with the `puppetdb` service.
+
+```ruby
+require "pmc_miller/reader"
+
+reader = PmcMiller::Reader.new("/path/to/puppet-metrics-collector")
+reader.service = :puppetdb
+data = reader.read("queue_depth")
```

Comment:
This was just my mental model of the object.  I thought it might provide preventing from having to pass the service for other methods that would require its context which would always be needed (e.g. reader.foo(service, bar)). If the `read` method is the only method, then it makes sense to add a service parameter.

The other option is to add the service to the `initialize` method so that it is configured at the time of creation.
```
db_reader = PmcMiller::Reader.new("/path/to/puppet-metrics-collector", :puppetdb)
srv_reader = PmcMiller::Reader.new("/path/to/puppet-metrics-collector", :puppetserver)
qd_data = db_reader.read("queue_depth")
free_jruby_data = srv_reader.read("average-free-jrubies")
```

Reply:
I don't think you want to add it to initialize, because then you have to create a reader per service... instead of one reader to read from any service.  And there isn't any reason to have more than one reader.  
And just in case my old suggestion of munging it with the key is in your head... don't... it was a bad idea... :)  having it as a param to the read seems like the best approach.

Reply:
The reader class has been refactored not to have a service attribute.  Thus the reader object is no longer tied to a particular service at any time.  The `read` method now takes a service and key to attempt to read.